src/EditorFeatures/Core/Shared/Tagging/TagProviders/AsynchronousViewTaggerProvider.cs

```diff
@@ -43,7 +61,7 @@ internal abstract class AbstractAsynchronousViewTaggerProvider<TTag> :
             return this.GetOrCreateTagger<T>(textView, subjectBuffer);
         }
 
-        internal sealed override bool TryRetrieveTagSource(ITextView textViewOpt, ITextBuffer subjectBuffer, out ProducerPopulatedTagSource<TTag> tagSource)
+        protected sealed override bool TryRetrieveTagSource(ITextView textViewOpt, ITextBuffer subjectBuffer, out ProducerPopulatedTagSource<TTag> tagSource)
```

Comment:
Not internal anymore because TagSources should not be exposed out of tagger providers.